civis/io/_tables.py

```diff
@@ -727,3 +762,48 @@ def f(x):
 
         fut.add_done_callback(f)
     return fut
+
+
+def _redshift_to_py(civ_client, tableid):
+
+    SQL_PANDAS_MAP = {
+        'smallint': np.int16,
+        'int': np.int32,
+        'integer': np.int32,
+        'bigint': np.int64,
+        'decimal': np.float64,
+        'numeric': np.float64,
+        'float': np.float64,
+        'float4': np.float32,
+        'float8': np.float64,
+        'real': np.float32,
+        'double precision': np.float64,
+        'boolean': np.bool,
+        'bool': np.bool,
+        'int2': np.int16,
+        'int4': np.int32,
+        'int8': np.int64,
+        'char': np.str,
+        'character': np.str,
+        'nchar': np.str,
+        'bpchar': np.str,
+        'varchar': np.str,
+        'character varying': np.str,
+        'nvarchar': np.str,
+        'text': np.str}
```

Comment:
@jseabold I think the type map here is pretty complete already. We just have to deal with nulls. Probs best to read as float64 and then do a quick check and convert back to the right in type if there are not any nulls.

Reply:
redshift by default uses UTF-8 encoding. Thus I think we should make sure to account for that and then punt on the rest.

Reply:
I've added a utf-8 encoding parameter to the kwargs